src/libtmux/session.py

```diff
@@ -259,6 +261,15 @@ def new_window(
             % (self.id, window_index),
         )
 
+        if environment:
+            if has_gte_version("3.0"):
+                for k, v in environment.items():
+                    window_args += ("-e%s=%s" % (k, v),)
```

Comment:
Bikeshed / fine to resolve:

This is what pyupgrade gives me

```suggestion
                    window_args += ("-e{}={}".format(k, v),)
```

You could also try f-strings

```suggestion
                    window_args += (f"-e{k}={v}",)
```

I am not sure if one is better than the other. 

You can change to f-string or keep it as-is and resolve